conf/schema.graphql

```diff
@@ -0,0 +1,59 @@
+type Consignment {
+  id: Int!
+  name: String!
+  series: Series!
+}
+
+input CreateFileInput {
+  path: String!
+  consignmentId: Int!
+  fileSize: Int!
+  lastModifiedDate: String!
+  fileName: String!
+  clientSideChecksum: String!
+}
+
+type File {
+  id: UUID!
+  path: String!
+  consignmentId: Int!
+  fileStatus: FileStatus!
+  pronomId: String
+  fileSize: Int!
+  lastModifiedDate: String!
```

Comment:
Is this the correct type for lastModifiedDate with the changes made to implement a date types?